keps/1145-additional-labels/README.md

```diff
@@ -0,0 +1,152 @@
+# KEP-1145: Additional labels, annotations, tolerations and selector
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories (Optional)](#user-stories-optional)
+    - [Story 1](#story-1)
+    - [Story 2](#story-2)
+  - [Notes/Constraints/Caveats (Optional)](#notesconstraintscaveats-optional)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+      - [Prerequisite testing updates](#prerequisite-testing-updates)
+    - [Unit Tests](#unit-tests)
+    - [Integration tests](#integration-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+Allow (AdmissionCheck)[https://github.com/kubernetes-sigs/kueue/tree/main/keps/993-two-phase-admission] 
+to set some additional location/admission information
+on workload's pods, so that they land on the machines of the `AdmissionCheck`
+choosing.
+
+## Motivation
+
+`AdmissionChecks` is a "plugin" mechanism for Kueue. Some of the plugins while 
+admitting may either narrow down on which machines the workload should start (by adding
+some extra node selector/tolerations) or add some tracking information
+that may be needed by other system components outside of Kueue.
+
+### Goals
+
+* Establish a mechanism through wich `AdmissionChecks` can add annotations/labels/node selector/tolerations
+  to workload pods.
+
+### Non-Goals
+
+* Allow `AdmissionChecks` to change pod definition in other way, for instance change command or image name.
+* Hard code any specific labels/annotations that are passed down.
+
+## Proposal
+
+* Add a struct in `Workload` status that allows to set annotations, labels, taints and tolerations on PodSets.
+* Expend PodSetInfo in Integration Framework to pass this information.
+* Modify existing integrations to support setting more data on the pods.
+
+### User Stories (Optional)
+
+#### Story 1
+
+My `AdmissionCheck` preallocates specific nodes (tainted and labeled) for a Workload to run, before it is 
+admitted (and creates any worker pods). Workload pods need to be modified so that they end up on correct nodes.
+
+#### Story 2
+
+My `AdmissionCheck` does budget checking and would like to mark pods that are possibly crossing budget with a 
```

Comment:
This isn't clear to me. Do we need the story? I suggest to either drop it, since story 1 provides already enough justification, or explain a little more the use case.

Reply:
This one justifies annotations and labels. For preallocated nodes you don't necessarily need these.